### ISSUE TITLE
Feature Request: Force show File names instead of Title / tags

### ISSUE THREAD

**Issue by joesphroth** [PaulWoitaschek/MaterialAudiobookPlayer#607]
I have posted this before and it got attached to other issues that were also closed. [#504](https://github.com/PaulWoitaschek/MaterialAudiobookPlayer/issues/504) [#433](https://github.com/PaulWoitaschek/MaterialAudiobookPlayer/issues/433)

I have some podcasts that do not have any tags so they all show as the same title and I can't distinguish between them.

Could you add a way to force display of file names?

**Comment by sandybeachit** [comment]
Why not just edit the files and add the tags or are all the podcasts your downloading missing tags ?

**Comment by joesphroth** [comment]
Yeah about half of the podcasts I download don't have episode title tags

**Comment by eMBee** [comment]
some podcast authors just don't set the titles on their files. if that happens then all files from that podcast are affected. to many to edit manually.

that said, puddletag is a useful tool to edit lots of audio files.

**Comment by deoren** [comment]
> @eMBee: that said, puddletag is a useful tool to edit lots of audio files.

Ex Falso is another good tool if you have find yourself needing to modify the tags yourself.

**Comment by thw0rted** [comment]
What is displayed when there aren't any tags, if not the file name? Seems like that would be a sensible fallback if the tags that are normally used are unavailable.

**Comment by Tim-Veestraeten** [comment]
I have the same problem but with podcasts that actually have tags. Opening the files in my default music player recognizes the title tags, but voice only shows podcast name (album) and hosts (artist), like it's going out of it's way to not show the title.
